src/modcdfnames.f90

```diff
@@ -276,6 +276,7 @@ SUBROUTINE ReadCdfNames ()
     READ(inam, namvars    )
     READ(inam, nambathy   )
     READ(inam, namsqdvar  )
+    READ(inam, namcubvar  )
```

Comment:
Is this something needed for your change or a bug you found when you worked on it ?